spec/asyncapi.md

```diff
@@ -647,6 +647,8 @@ subscribe:
 
 Describes a publish or a subscribe operation. This provides a place to document how and why messages are sent and received. For example, an operation might describe a chat application use case where a user sends a text message to a group.
 
+A publish operation describes messages that are received by the application, whereas a subscribe operation describes messages that are sent by the application.
```

Comment:
```suggestion
A publish operation describes messages that are received by the chat application, whereas a subscribe operation describes messages that are sent by the chat application.
```

how about this and not as a separate paragraph but continuation of previous paragraph